packages/desktop-client/src/components/App.tsx

```diff
@@ -157,6 +158,10 @@ function ErrorFallback({ error }: FallbackProps) {
 }
 
 export function App() {
+  const store = useStore();
+
+  useEffect(() => handleGlobalEvents(store), [store]);
```

Comment:
Moved call to handleGlobalEvents here so that we can have proper cleanup of listeners